store/store.go

```diff
@@ -26,14 +26,15 @@ const (
 	versionCollectionName        string = "VersionCollection"
 	versionLogCollectionName     string = "VersionLogCollection"
 	recourceCollectionName       string = "RecourceCollection"
-	projectCollectionName        string = "ProjectCollection"
 	projectVersionCollectionName string = "ProjectVersionCollection"
 	daemonCollectionName         string = "DaemonCollection"
 	remoteCollectionName         string = "RemoteCollection"
 	workerNodeCollection         string = "WorkerNodeCollection"
 	deployCollectionName         string = "DeployCollection"
 	ResourceCollectionName       string = "ResourceCollection"
 	CloudsCollection             string = "clouds"
+
+	projectCollectionName string = "projects"
```

Comment:
The `project` had been used for union building. We need to clean the old code in another PR